tests/e2e/tests-library/ProjectAndFileTests.ts

```diff
@@ -60,4 +63,43 @@ export class ProjectAndFileTests {
 			Logger.info('Second welcome content dialog box was not shown');
 		}
 	}
+
+	async getProjectViewSession(): Promise<ViewSection> {
+		Logger.debug();
+
+		await this.driverHelper.waitVisibility(
+			this.cheCodeLocatorLoader.webCheCodeLocators.DefaultTreeSection.itemRow,
+			TIMEOUT_CONSTANTS.TS_EXPAND_PROJECT_TREE_ITEM_TIMEOUT
+		);
+
+		const viewContent: ViewContent = new SideBarView().getContent();
+		const [projectSection]: ViewSection[] = await viewContent.getSections();
+		return projectSection;
+	}
+
+	async getProjectTreeItem(projectSection: ViewSection, item: string, itemLevel: number = 2): Promise<ViewItem | undefined> {
```

Comment:
"magic number" is found :-) 

is it project root level, or multi-project root level, or sub-directory level?

What do you think about extracting `2` as a constant with clear name?

Reply:
itemLevel: number = 2 -  for me, it is obvious it is not literal and not magic number

Reply:
@musienko-maxim : how do you know which level it starts from: 0 or 1?

Reply:
it is max level inside project tree where findItem() will try to find item 
    
    /**
     * Find an item in this view section by label. Does not perform recursive search through the whole tree.
     * Does however scroll through all the expanded content. Will find items beyond the current scroll range.
     * @param label Label of the item to search for.
     * @param maxLevel Limit how deep the algorithm should look into any expanded items, default unlimited (0)
     * @returns Promise resolving to ViewItem object is such item exists, undefined otherwise
     */
    abstract findItem(label: string, maxLevel?: number): Promise<ViewItem | undefined>;

Reply:
I added info as JSDoc

Reply:
Thanks for adding docs to the code, @nallikaea !
It's now clear to me.